src/app/dropdown-button/dropdown-button.component.ts

```diff
@@ -132,6 +131,7 @@ export class DropdownButtonComponent implements OnChanges {
     }
     this.selectedActionOption = option;
     this.toggleActionOptionsMenu(false);
+    this.selectActionOptionEvent.next(this.selectedActionOption);
```

Comment:
You should use emit() here instead of next(). next() is not part of the official API (https://angular.io/api/core/EventEmitter) and is dependent on the implementation of EventEmitter, which could potentially change at some point in the future,

Reply:
That's good to know. Changed to `emit()`.